packages/flame_tiled/lib/src/renderable_tile_map.dart

```diff
@@ -192,33 +192,55 @@ class RenderableTiledMap {
   /// Parses a file returning a [RenderableTiledMap].
   ///
   /// NOTE: this method looks for files under the path "assets/tiles/".
+  ///
+  /// By default, [FlameTileLayer] renders flipped tiles if they exist.
+  /// You can disable it by passing [ignoreFlip] as `true`.
```

Comment:
```suggestion
  /// By default, [FlameTileLayer] renders flipped tiles if they exist.
  /// You can disable this by setting [ignoreFlip] to `true`.
```

We should turn this into a Dart doc template and reuse that on the other parts as well to prevent doc duplication.

Reply:
Cool! I didn't know this is possible.